### PR TITLE
Feature/details

### PR DESCRIPTION
**Pull request by ReBlack**
## Summary

(select one:)
* Breaking change
* **Addition**
* Patch for #
* New provider

- [ +] Tested
  - [ +] Unit tests
  - [ +] Style
- [ +] Documentation (PR # )

## Goal

Details functional in nominatim

## Description

Add details by placeid
Add details by osmtype and osmid

### CODE REVIEW

**Comment by maxhelias** [comment]
Thanks @ReBlack for your contribution 🍻

File: src/Details.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    /**
+     * Place information by osmtype and osmid.
+     *
+     * @return Details
+     */
+    public function osmId(string $osmType, int $osmId): self
+    {
+        $this->query['osmtype'] = $osmType;
```
Comment (maxhelias) [inline src/Details.php:55]: I will update this method to split osmtype & osmid in 2 method as for the "Reverse" provider

Comment (ReBlack) [reply src/Details.php:55]: ok, thanks. Sorry )

**Comment by maxhelias** [comment]
Please see #12

**Comment by maxhelias** [comment]
I'm also thinking about removing the generated phpdoc, I'm not sure this is useful and the annotation should be sufficient

**Comment by ReBlack** [comment]
I agree

**Pull request merged**